assets/js/mbta.ts

```diff
@@ -3135,15 +3169,19 @@ const arincToRealtimeIdMap: { [key: string]: string } = {
   'GSCI-e': 'science_park_eastbound',
   'GSCI-w': 'science_park_westbound',
   'GSCI-m': 'science_park_mezzanine',
-  'GLEC-w': 'lechmere_green_line',
+  'GLEC-e': 'lechmere_green_line_eastbound',
+  'GLEC-w': 'lechmere_green_line_westbound',
+  'GLEC-m': 'lechmere_green_line_mezzanine',
+  'GUNS-e': 'union_sq_track_one',
+  'GUNS-w': 'union_sq_track_two',
+  'GUNS-m': 'union_sq_mezzanine',
   'SDUD-n': 'bus.Nubian_Platform_F',
   'SDUD-s': 'bus.Nubian_Platform_C',
   'SDUD-e': 'bus.Nubian_Platform_D',
   'SDUD-w': 'bus.Nubian_Platform_A',
   'SDUD-c': 'bus.Nubian_Platform_E_east',
   'SDUD-m': 'bus.Nubian_Platform_E_west',
-  'GLEC-n': 'bus.Lechmere_bus_north',
-  'GLEC-m': 'bus.Lechmere_bus_south',
+  'SLEC-m': 'bus.Lechmere_bus_mezzanine',
```

Comment:
Did you talk to Dan Salomon about these changes? These IDs come from TransitWay Engine which he maintains.

Reply:
Not yet - will reach out and make sure we're on the same page about the ids!

Reply:
Ultimately this has to agree with ARINC's zone assignments. If the plan is to have the busway be SLEC m, this is fine with me, and I'll configure the Transitway Engine to agree with this. Will there also be sign(s) in the GLEC m zone for rail predictions?

Are these configured as `auto: false,` and `headway: false,` to lock off the signs because revenue service hasn't started?

Reply:
That's correct - SLEC-m is for buses and there is also GLEC-m for rail.

And yep, the plan currently is to follow up and unlock those signs for `headway` and `auto` mode once revenue service begins